contents/handbook/growth/sales/health-tracking.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Customer health scoring
```

Comment:
This is all good stuff, but I'm slightly confused by the distinction between health score and risk indicators - as an AE, I just track both? Or is it more that risk indicators are 'this is an alarm, you need to respond' vs. health score is 'this is just a general picture of how things are going'?

Reply:
Yep - just called that our more explicitly in the indicator section